### PR TITLE
add blog

### PR DESCRIPTION
**Pull request by GreyPlane**
Fixes # .

Make sure these boxes are checked before creating your pull request -- thank you!

- [ ] Add Blog Header, including title, author, date.
e.g.
```
---
 
 title: "'Score Your Face Photo' a ML&Wechaty practice"
 date: 2017-09-18 09:00 +0800
 author: huyingxi
 ---
 
 > Author: [@huyingxi](https://github.com/huyingxi/wechaty_selfie) enjoying ML&Wechaty at BUPT
```
- [x] Keep all filenames & url as lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor` instead of `2017-10-06-WeChat PC Impactor`,
- [ ] Embed the photo & video before publishing, save all external file to the blog `/download/2018` directory.
- [ ] Add `<!--more-->` section to just show the abstract in blog homepage.
- [ ] Add teaser image for the blog
- [ ] Add your self as an author to the author data file at https://github.com/Chatie/blog/blob/master/docs/_data/authors.yml with your GitHub username. And please also remember to reference yourself with the Github username in your blog post, so that we can have nice author information besides your blog after we published it!

[See more](https://github.com/Chatie/blog#contribute-guideline)

### CODE REVIEW

**Comment by lijiarui** [comment]
@GreyPlane Thanks for the post.

Please do the following things and then I can merge this blog:

- [ ] Add teaser image for the blog
- [x] Add your self as an author to the author data file at https://github.com/Chatie/blog/blob/master/docs/_data/authors.yml with your GitHub username. And please also remember to reference yourself with the Github username in your blog post, so that we can have nice author information besides your blog after we published it!

**Comment by lijiarui** [comment]
@GreyPlane Thanks for your update, please add a teaser image for the blog at the header.

**Comment by lijiarui** [comment]
ping @GreyPlane

File: _posts/2020-04-15-nakano-bot-blog.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+author: GreyPlane
+
+date: '2020-04-15 09:30:00 +0800'
+
+categories: project
+
+tags: - wechaty
+      - selenium
+
```
Comment (lijiarui) [inline _posts/2020-04-15-nakano-bot-blog.md:13]: Please add a teaser for this blog, it can make this blog more beautiful, thanks!

**Review by lijiarui — CHANGES_REQUESTED** [review]
Please change as I request.

**Comment by lijiarui** [comment]
Also, it would be nice if you can resolve the confilct.

**Comment by GreyPlane** [comment]
> 
> 
> Also, it would be nice if you can resolve the confilct.

sry for take so long to reply, i'm pretty busy recently, i'll fix it as soon as possible

**Comment by lijiarui** [comment]
ping @GreyPlane

**Comment by lijiarui** [comment]
ping @GreyPlane 

Please let me know when will you fix this PR, or I will close this PR at the end of June.

**Pull request closed without merging**